src/components/ListView.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import ListCard from '../components/ListCard';
+import { withStyles } from '@material-ui/core/styles';
+import axios from 'axios';
+import CircularProgress from '@material-ui/core/CircularProgress';
+
+const styles = theme => ({
+  container: {
+    backgroundColor: 'grey',
+    display: 'flex',
+    alignItems: 'center',
+    flexDirection: 'column',
+    height: '100%',
+    justifyContent: 'center'
+  }
+});
+
+class ListView extends React.Component {
+
+  state = {reports: null};
+
+  componentDidMount() {
+    axios.get(process.env.REACT_APP_GET_REPORTS)
```

Comment:
Let's also remove this from the `.env` file and then just define a const outside the class where the cloud function lives on Firebase